monai/transforms/intensity/array.py

```diff
@@ -1603,3 +1606,61 @@ def _to_numpy(self, img: Union[np.ndarray, torch.Tensor]) -> Tuple[np.ndarray, t
             return img.cpu().detach().numpy(), img.device
         else:
             return img, torch.device("cpu")
+
+
+class RandCoarseDropout(RandomizableTransform):
+    """
+    Randomly coarse dropout regions in the image, then fill in the "holes" with specified value.
```

Comment:
-> "Randomly coarse dropout rectangular regions in the image, then fill in the "holes" with a specified value."

Should emphasize that it's rectangular regions being filled in.

Reply:
Sure, updated in the latest commit.
Thanks.